dom/dom_c.php

```diff
@@ -1720,7 +1720,7 @@ class DOMElement extends DOMNode implements DOMParentNode, DOMChildNode
     public $nextSibling;
 
     /**
-     * @var DOMNamedNodeMap<DOMAttr>
+     * @var DOMNamedNodeMap<DOMAttr>|null
```

Comment:
The type also need to be updated in the `LanguageLevelTypeAware` attribute defining the native type per PHP version.